hack/windows/PrepareServiceInterface.ps1

```diff
@@ -0,0 +1,29 @@
+<#
```

Comment:
This file is for testing only too?

Reply:
@jianjuns I think this script can be provided to user. Or we need describe the content in document.  Do you think it's the proper location for this file?

Reply:
Then I think it should be together with InstallOVS.ps1. Feel a dir like /dist might be better for files to publish, but since we do not have many cases now, I am file to keep them in /hack.